templates/managed/apps/kafka.html

```diff
@@ -13,8 +13,7 @@
       <h1>
         Managed Apache Kafka
       </h1>
-      <p><i>Now running Kafka 2.8</i></p>
-      <p class="p-heading--4">Fully managed Apache Kafka anywhere, on-prem and multi-cloud environments.</p>
+      <p class="p-heading--4">Fully managed Apache Kafka anywhere: on-prem and multi-cloud environments.</p>
       <p>Spend less time managing infrastructure, more time building applications. Highly available, secure and fully managed Apache Kafka for frictionless innovation.</p>
```

Comment:
This sentence is deleted on the doc

Reply:
It is actually still there. The sentence was erroneously duplicated below this section and is removed from the copy doc in the duplicated location.